src/zenml/cli/hub.py

```diff
@@ -0,0 +1,1164 @@
+
+        # Check if plugin can be installed
+        index_url = plugin.index_url
+        package_name = plugin.package_name
+        if not index_url or not package_name:
+            error(
+                f"Plugin '{display_name}' is not available for installation."
+            )
+
+        # Check if plugin is already installed
+        if _is_plugin_installed(plugin) and not upgrade:
+            logger.info(f"Plugin '{plugin_name}' is already installed.")
+            return
+
+        # Install plugin requirements
+        install_requirements = False
+        if plugin.requirements and not no_deps:
+            requirements_str = " ".join(f"'{r}'" for r in plugin.requirements)
+            if not yes:
+                install_requirements = click.confirm(
+                    f"Plugin '{display_name}' requires the following "
+                    f"packages to be installed: {requirements_str}. "
+                    f"Do you want to install them now?"
+                )
+        if plugin.requirements and install_requirements:
+            logger.info(
+                f"Installing requirements for plugin '{display_name}': "
+                f"{requirements_str}..."
+            )
+            requirements_install_call = [
+                sys.executable,
+                "-m",
+                "pip",
+                "install",
+                *list(plugin.requirements),
+                "--upgrade",
+            ]
+            subprocess.check_call(requirements_install_call)
+            logger.info(
+                f"Successfully installed requirements for plugin "
+                f"'{display_name}'."
+            )
+        elif plugin.requirements:
+            logger.warning(
+                f"Requirements for plugin '{display_name}' were not installed. "
+                "This might lead to errors in the future if the requirements "
+                "are not installed manually."
+            )
+
+        # pip install the wheel
+        logger.info(
+            f"Installing plugin '{display_name}' from "
+            f"{index_url}{package_name}..."
+        )
+        plugin_install_call = [
+            sys.executable,
+            "-m",
+            "pip",
+            "install",
+            "--index-url",
+            index_url,
+            package_name,
+            "--no-deps",  # we already installed the requirements above
+            "--upgrade",  # we already checked if the plugin is installed above
+        ]
+        subprocess.check_call(plugin_install_call)
+        logger.info(f"Successfully installed plugin '{display_name}'.")
+
+
+@hub.command("uninstall")
+@click.argument("plugin_name", type=str, required=True)
+@click.option(
+    "--version",
+    "-v",
+    type=str,
+    help="Version of the plugin to uninstall.",
+)
+def uninstall_plugin(
+    plugin_name: str,
+    version: Optional[str] = None,
+) -> None:
+    """Uninstall a plugin from the hub.
```

Comment:
I think the docstring is a bit misleading here. This command uninstalls it from your environment, not the hub.